### ISSUE TITLE
CAPTCHA fixes

### ISSUE THREAD

**Issue by harrislapiroff** [freedomofpress/securedrop.org#455]
- [x] Remove recaptcha from the site (it is currently being blocked by our CSP and we don't wanna be pinging Google anyway)
- [ ] Look into implementing `django-captcha` on the site. If it's trivially easy, do it this week. Otherwise let's keep a ticket open for that down the road.

**Comment by harrislapiroff** [comment]
[Backlog pruning 5/10] Duplicate of #452
